codenames-domain/src/main/java/com/codenames/mapper/RoomMapper.java

```diff
@@ -0,0 +1,40 @@
+package com.codenames.mapper;
+
+//import org.mapstruct.Mapper;
+//import org.mapstruct.Mapping;
+
+import com.codenames.dto.RoomDto;
+import com.codenames.dto.TeamDto;
+import com.codenames.dto.UserDto;
+import com.codenames.dto.WordDto;
+import com.codenames.models.forgame.Player;
+import com.codenames.models.forooms.Room;
+
+import java.util.List;
+
+//@Mapper
```

Comment:
Better go on with using mapstruct
It makes code free of boilerplate mappers like below